custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -824,12 +873,20 @@ def _maybe_build_image(self, docker_image_or_directory):
                         "Failed to build a docker image:\n{}".format(all_lines)
                     )
 
-                print("\nImage successfully built; tag: {}; image id: {}\n".format(tag, image_id))
+                print("\nImage successfully built; tag: {}; image id: {}".format(tag, image_id))
+                print(
+                    "It is recommended to use --docker {}, if you don't need to rebuild the image.\n".format(
+                        tag
```

Comment:
how do you get here without using --docker?

Reply:
What do you mean?
This message is shown only when you provide a context folder:
` --docker docker_context_dir/my_cool_env`

So in the end I say: hey, now you have the image tagged as `my_cool_env` use `--docker my_cool_env` for the next time.

Reply:
got it , awesome